scripts/windows/installs/vs2013.ps1

```diff
@@ -1,4 +1,19 @@
-(New-Object System.Net.WebClient).DownloadFile('https://archive.org/download/msdn_2023/en_visual_studio_express_2013_for_windows_desktop_with_update_4_x86_dvd_5920567.iso', 'C:\Windows\Temp\vs2013express.iso')
```

Comment:
I migrated away from this URL as it was going to take >2 hours for me to install 😄